checklists.md

```diff
@@ -56,7 +56,6 @@ https://www.npr.org/templates/story/story.php?storyId=122226184 <br />
 [Shane Parrish](https://fs.blog/2014/02/the-checklist-manifesto),
 [Malcolm Gladwell](http://atulgawande.com/book/the-checklist-manifesto),
 [Vinod Khosla](https://twitter.com/vkhosla/status/222469073806700544),
-[Bill Gates](https://www.gatesnotes.com/Books/Checklist-Manifesto), etc.
```

Comment:
Unfortunately the link returns a 404 page and looking for the book doesn't return any matching result:
![image](https://user-images.githubusercontent.com/6057298/68589918-1c4a1500-0485-11ea-9626-9223b83d0bcd.png)

Reply:
https://www.analogue.network/book/the-checklist-manifesto-wp3t87?u=billgates I think has it

Reply:
Indeed the existing link appears to **`404`**:
![image](https://user-images.githubusercontent.com/194400/68590528-a777da80-0486-11ea-8c7d-b455b8a5d15c.png)

@iteles thanks for tracking down an alternative link. 👍

Reply:
done :arrow_down: